30-remote-site-setup.sh

```diff
@@ -66,20 +65,24 @@ echo $REMOTE_HOST_KEY >> $root_ssh_dir/known_hosts
 # Populate the bosco override dir from a Git repo
 GIT_SSH_KEY=/etc/osg/git.key
 [[ -f $GIT_SSH_KEY ]] && export GIT_SSH_COMMAND="ssh -i $GIT_SSH_KEY"
-[[ -z $BOSCO_GIT_ENDPOINT || -z $BOSCO_DIRECTORY ]] || \
+if [[ -z $BOSCO_GIT_ENDPOINT || -z $BOSCO_DIRECTORY ]]; then
+    OVERRIDE_DIR=/etc/condor-ce/bosco_override
     /usr/local/bin/bosco-override-setup.sh "$BOSCO_GIT_ENDPOINT" "$BOSCO_DIRECTORY"
+fi
 unset GIT_SSH_COMMAND
 
 users=$(cat /etc/grid-security/grid-mapfile /etc/grid-security/voms-mapfile | \
             awk '/^"[^"]+" +[a-zA-Z0-9\-\._]+$/ {print $NF}' | \
             sort -u)
 [[ -n $users ]] || exit 1
 
+override_opts=""
 for ruser in $users; do
     setup_ssh_config
     setup_endpoints_ini
+    [[ -d $OVERRIDE_DIR ]] && override_opts="-o $OVERRIDE_DIR"
```

Comment:
Small detail here, but i guess there is no warning if `OVERRIDE_DIR` is set, but to a non-existing directory

Reply:
Fixed